toolkit/sharedlib/src/main/java/com/arcgismaps/toolkit/ui/expandablecard/theme/DefaultThemeTokens.kt

```diff
@@ -0,0 +1,157 @@
+/*
+ * Copyright 2024 Esri
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+@file:Suppress("unused")
+
+package com.arcgismaps.toolkit.ui.expandablecard.theme
+
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.text.PlatformTextStyle
+import androidx.compose.ui.text.TextStyle
+import androidx.compose.ui.text.font.FontFamily
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import com.arcgismaps.toolkit.ui.expandablecard.theme.DefaultThemeTokens.colorScheme
+import com.arcgismaps.toolkit.ui.expandablecard.theme.DefaultThemeTokens.typography
+
+/**
+ * Provides a default [ExpandableCardColorScheme] via [colorScheme] and a default [ExpandableCardTypography]
+ * via [typography].
+ *
+ * Creates a default theme from statically defined tokens. Do not use this at runtime, instead the
+ * ExpandableCardTheme must always be specified from the current MaterialTheme.
+ *
+ * This mainly provides a default theme for use within previews.
+ *
+ */
+internal object DefaultThemeTokens {
+
+    /**
+     *  The default values provided are hardcoded based on the material 3 light color scheme.
+     *  See [androidx.compose.material3.lightColorScheme].
+     */
+    val colorScheme: ExpandableCardColorScheme = ExpandableCardColorScheme(
+        headerTextColor = ColorTokens.OnSurface,
+        containerColor = ColorTokens.PrimaryContainer,
+        galleryContainerColor = ColorTokens.OnBackground,
+        borderColor = ColorTokens.Outline.copy(alpha = 0.6f)
+    )
+
+    /**
+     * The default values provided are hardcoded based on the material 3 typography.
+     * See [androidx.compose.material3.Typography]
+     */
+    val typography: ExpandableCardTypography = ExpandableCardTypography(
+        headerStyle = TypographyTokens.bodyLarge,
+        bodyStyle = TypographyTokens.bodyMedium
+    )
+
```

Comment:
Just for consistency
```suggestion

    /**
     * The default values provided are hardcoded based on the material 3 shapes.
     * See [androidx.compose.material3.Shapes]
     */
```